sns/parity_defs.h

```diff
@@ -26,7 +26,11 @@
 /**
  *  Define the condition here to use Intel ISA library.
  */
-#define ISAL_ENCODE_ENABLED	(!defined(__KERNEL__) && defined(HAVE_ISAL))
+#if (!defined(__KERNEL__) && defined(HAVE_ISAL))
```

Comment:
Style (minor): outer parenthesises are redundant here. Consider changing to:
```
#if !defined(__KERNEL__) && defined(HAVE_ISAL)
```
It makes it more readable also.